src/index/LocatedTriples.h

```diff
@@ -0,0 +1,174 @@
+// Copyright 2023 - 2024, University of Freiburg
+// Chair of Algorithms and Data Structures
+// Authors:
+//    2023 Hannah Bast <bast@cs.uni-freiburg.de>
+//    2024 Julian Mundhahs <mundhahj@tf.uni-freiburg.de>
+
+#pragma once
+
+#include "engine/idTable/IdTable.h"
+#include "global/IdTriple.h"
+#include "index/CompressedRelation.h"
+#include "util/HashMap.h"
+
+class Permutation;
+
+IdTriple permute(const IdTriple& triple, const std::array<size_t, 3>& keyOrder);
```

Comment:
This could also indicate that `IdTriple` should be its own type instead of an alias. This was an TODO at the alias definition but seems to have got lost.